spec/requests/articles/articles_update_spec.rb

```diff
@@ -167,4 +167,43 @@
     expect(response).to redirect_to "#{article.path}/edit"
     expect(article.reload.video_thumbnail_url).to include "https://i.imgur.com/HPiu7N4.jpg"
   end
+
+  context "when main_image_from_frontmatter is false" do
```

Comment:
These are great specs, I'm wondering if it would be better to put these as model tests.  The request specs spin up a lot of context to run, as compared to model specs.

Thoughts?

Reply:
for sure they can be moved. Do you think its worth keeping the once which directly resolves the issue. But to be fair that can be moved too.

Reply:
My personal preference is to have that be a model test.  Those request tests are slow, and if we already cover the issue with a model test, I'd like to avoid adding a slow test to the suite.

Reply:
@jeremyf done as required